io.github.ptitSeb.hydracastlelabyrinth.yml

```diff
@@ -0,0 +1,39 @@
+app-id: io.github.ptitSeb.hydracastlelabyrinth
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: io.github.ptitSeb.hydracastlelabyrinth.sh
+
+finish-args:
+  # hardware 3D
+  - --device=dri
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
It's either x11 or (wayland and fallback-x11). Both does not make sense

Reply:
That's what I said 

see https://github.com/flathub/flathub/pull/4928#discussion_r1481517031

Reply:
Ah, apologies - I'm not super familiar with how Wayland works so I completely misunderstood how these permissions work. Sorted now!